### PR TITLE
update packages/devel

### PR DESCRIPTION
**Pull request by heitbaum**
- cmake: update to 3.24.0
- elfutils: update to 0.187
- glib: update to 2.73.3
- glib: update to pcre2 instead of pcre as a dependancy
- go: update to 1.18.5
- libbpf: update to 0.8.1
- libcap: update to 2.65
- pcre2: move from addons to devel directory
- slang: update to 2.3.3

### CODE REVIEW

**Comment by SupervisedThinking** [comment]
If pcre2 is now a hard dependeny it probably should be moved to packages/devel where pcre is already located

**Comment by heitbaum** [comment]
> If pcre2 is now a hard dependeny it probably should be moved to packages/devel where pcre is already located

Good call - moved.

File: packages/devel/cmake/package.mk [hunk 1]
```diff
@@ -3,11 +3,12 @@
...
 
 PKG_NAME="cmake"
-PKG_VERSION="3.23.2"
-PKG_SHA256="f316b40053466f9a416adf981efda41b160ca859e97f6a484b447ea299ff26aa"
+PKG_VERSION="3.24.0"
+PKG_SHA256="c2b61f7cdecb1576cad25f918a8f42b8685d88a832fd4b62b9e0fa32e915a658"
 PKG_LICENSE="BSD"
 PKG_SITE="https://cmake.org/"
 PKG_URL="https://cmake.org/files/v$(get_pkg_version_maj_min)/cmake-${PKG_VERSION}.tar.gz"
+PKG_URL="https://cmake.org/files/v3.24/cmake-${PKG_VERSION}.tar.gz"
```
Comment (CvH) [inline packages/devel/cmake/package.mk:11]: wrong url :)

Comment (heitbaum) [reply packages/devel/cmake/package.mk]: > wrong url :)

Thanks for spotting that. (Joys of tracking RCs and squashing.)

**Review by CvH — APPROVED** [review]

**Pull request merged**